### DISCUSSION TITLE
Simulation Error: Events ['Maximum voltage [V]'] are non-positive at initial conditions

### DISCUSSION THREAD

**Discussion by Dhruv-RIL** [pybamm-team/PyBaMM#4030] [Q&A]
Hi,
I am trying to simulate an LFP cell using a modified Prada parameter set.
However, while solving the model it is throwing an error **Events ['Maximum voltage [V]'] are non-positive at initial conditions**
I am not able to get around this error, could someone help me?
Thanks in advance

**Comment by martinjrobins** [comment]
Hi @Dhruv-RIL, the models have a maximum voltage cut-off event that triggers when the terminal voltage exceeds a threshold set in the parameter set. The way you have setup your model means that the initial terminal voltage (i.e. at t = 0) is above this threshold. We would be able to give you more specific advice if you included your python script that gives you this error, and the full error message.

**Reply by Dhruv-RIL** [reply]
Sure! I have attached my script and the error message.
[LFP_ModelScript.pdf](https://github.com/pybamm-team/PyBaMM/files/15087913/LFP_ModelScript.pdf)

**Reply by Dhruv-RIL** [reply]
Hi @martinjrobins, I am not able to get past the errors.
Can you look at the script and suggest what is going wrong?
